spinalcordtoolbox/registration/register.py

```diff
@@ -1234,7 +1234,7 @@ def find_index_halfmax(data1d):
 
 def find_angle_hog(image, centermass, px, py, angle_range=10):
     """
-    Finds the angle of an image based on the method described by Sun, “Symmetry Detection Using Gradient Information.”
+    Finds the angle of an image based on the method described by Sun, 'Symmetry Detection Using Gradient Information.'
```

Comment:
```suggestion
    Finds the angle of an image based on the method described by Sun, "Symmetry Detection Using Gradient Information."
```

Reply:
Ah right. Using `'` in strings is so habitual that I didn't notice this was triple-quoted. :p